src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -520,6 +531,80 @@ export function downSampling(
     });
 }
 
+export function floorScatterData(
+    scatterDataWithOpacity: ScatterData[]
+): ScatterData[] {
+    const eventInfoByMonth: { [month: number]: EventInfo } = {};
+    let eventInfo: EventInfo;
+    let result: ScatterData[] = [];
+    _.reduce(
```

Comment:
you should always assign to the variable with any map/reduce/filter/etc operation.
```
const eventInfoByMonth = _.reduce(input, (aggr:TYPE HERE)=> ... })
```

Reply:
updated